teoPalomino/budget-lens-backend
# Feat/bud 49 Default Pagination Reciepts

Pagination of Receipts

To test:
- Go to `api/receipts/<pageNumber>/<pageSize>` where `pageNumber` and `pageSize` are replaced by integer values.
- Make sure user is authorized. Use postman for that.

~~_Note: I forgot about the Merchant model. Let me add that before merging._~~
